.github/workflows/ros-ci.yml

```diff
@@ -14,8 +15,18 @@ jobs:
           - {ROS_DISTRO: melodic}
           - {ROS_DISTRO: noetic}
     env:
-      CCACHE_DIR: /github/home/.ccache # Enable ccache
+      CCACHE_DIR: /github/home/.ccache             # Enable ccache
+      UPSTREAM_WORKSPACE: dependencies.rosinstall  # to build example-robot-data from source as it's not released via the ROS buildfarm
```

Comment:
```suggestion
```

The build will fail since there is no `dependencies.rosinstall` in the repository. We also don't need it since the only dependencies Pinocchio requires (EigenPy) are available via the ROS buildfarm